calculator.rb

```diff
@@ -0,0 +1,114 @@
+def make_numeric(num)
+  if num.include?('.')
+    num = num.to_f
+  else
+    num = num.to_i
+  end
+end
+
+def number?(obj)
+  obj = obj.to_s unless obj.is_a? String
+  /\A[+-]?\d+(\.[\d]+)?\z/.match(obj)
+end
+
+def rounded(result)
+  if result.to_s.include?('.')
+    return result.round(2)
+  else
+    return result
+  end
+end
+
+def add(num_1, num_2)
+  result = num_1 + num_2
+  return rounded(result)
+end
+
+def subtract(num_1, num_2)
+  result = num_1 - num_2
+  return rounded(result)
+end
+
+def multiply(num_1, num_2)
+  result = num_1 * num_2
+  return rounded(result)
+end
+
+def divide(num_1, num_2)
+  num_1 = num_1.to_f
+  num_2 = num_2.to_f
+  result = num_1 / num_2
+
+  if num_2 == 0
+    puts "Does Not Exist"
+  else
```

Comment:
I like that you've included the division-specific error handling code in this method.